styles.css

```diff
@@ -0,0 +1,69 @@
+body {
+  min-width: 375px;
+  margin: 0;
+  padding: 0;
+
+  /* box-sizing: border-box; */
+}
+
+.menu {
+  width: 1.25rem;
+  margin-left: 17.75rem;
+  margin-top: 1.188rem;
+}
+
+header {
+  background-image: url(./icons/image_geometry_1.png);
+  background-repeat: no-repeat;
+  height: 4.625rem;
+  display: flex;
+  align-items: center;
+}
+
+.homepage {
+  width: 20rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+}
+
+h1 {
+  font-family: Playfair Display, arial, sans-serif;
+  font-size: 50px;
+  font-weight: 400;
+}
+
+.text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: -1.25rem 2rem 1rem 2rem;
+}
+
+.text p {
+  font-family: Poppins, arial, sans-serif;
+  font-size: 0.875rem;
+  line-height: 1.313rem;
+  letter-spacing: -0.01rem;
+}
+
+.collaboration {
+  width: 14rem;
+  height: 3.125rem;
+  margin: 1rem 2rem;
+  background-color: #1a2236;
+  color: white;
+  font-family: Poppins, arial, sans-serif;
+  font-size: 1rem;
+  font-weight: 500;
+  line-height: 1.5rem;
+  letter-spacing: 0;
+  text-align: center;
+  cursor: pointer;
+}
```

Comment:
- [ ] It would be better if you consider adhering to the `Figma` design in this part of your design as you have in most parts of your design. Kindly make sure that your `button` is aligned on the same starting point as the content of your page as shown in the `Figma` design. See below for more details:

  <table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="316" alt="Screenshot 2023-07-05 at 19 11 02" src="https://github.com/JohnteFresh/IrunguJohnson-Mobile-Portfolio/assets/107192076/7823ce11-e7ed-4a55-a14f-fc0b2c730673">
</td>
<td>
<img width="316" alt="Screenshot 2023-07-05 at 19 11 16" src="https://github.com/JohnteFresh/IrunguJohnson-Mobile-Portfolio/assets/107192076/0f650946-79d8-4706-8b32-21c83a4fec98">
</td>
</tr>
</table>